Unix/installbuilder/datafiles/AIX.data

```diff
@@ -74,3 +87,15 @@ ${{SERVICE_SCRIPT}} start
 #include OmiService_funcs
 StopOmiService
 RemoveOmiService
+
+%Postuninstall_100
+egrep "^omi:" /etc/passwd 2>&1 >/dev/null
+if [ $? -eq 0 ]; then
+    echo "Deleting omi service account ..."
+    userdel omi
+fi
+egrep "^omi:" /etc/group 2>&1 >/dev/null
+if [ $? -eq 0 ]; then
+    echo "Deleting omi group ..."
+    rmgroup omi
+fi
```

Comment:
Yes, this is a problem on Linux. On AIX/HP/Solaris, you should be okay.

Check https://github.com/Microsoft/PowerShell-DSC-for-Linux/blob/master/installbuilder/datafiles/Base_DSC.data on line 317 for an example of how to check (for Linux) if this is an upgrade call to postuninstall, or not.